.travis.yml

```diff
@@ -1,13 +1,26 @@
 language: java
-jdk:
-- oraclejdk8
 sudo: false
+# see https://sormuras.github.io/blog/2018-03-20-jdk-matrix.html
+matrix:
+  include:
+    # 8
+    - env: JDK='Oracle JDK 8'
```

Comment:
Is it worth testing against both Oracle and OpenJDK?  Don't imagine we will need to worry about compatbility issues between the two.

Reply:
OK, I will disable one of them.